bolts/data/datasets/vision/celeba.py

```diff
@@ -21,68 +24,158 @@
 )
 from bolts.data.structures import TernarySample
 
-__all__ = ["Celeba"]
+__all__ = ["CelebA", "CelebAttr", "CelebASplit"]
 
 LOGGER = logging.getLogger(__name__)
 
 
-class Celeba(VisionDataset):
+class CelebAttr(Enum):
+    Five_o_Clock_Shadow = auto()
+    Arched_Eyebrows = auto()
+    Attractive = auto()
+    Bags_Under_Eyes = auto()
+    Bald = auto()
+    Bangs = auto()
+    Big_Lips = auto()
+    Big_Nose = auto()
+    Black_Hair = auto()
+    Blond_Hair = auto()
+    Blurry = auto()
+    Brown_Hair = auto()
+    Bushy_Eyebrows = auto()
+    Chubby = auto()
+    Double_Chin = auto()
+    Eyeglasses = auto()
+    Goatee = auto()
+    Gray_Hair = auto()
+    Heavy_Makeup = auto()
+    High_Cheekbones = auto()
+    Male = auto()
+    Mouth_Slightly_Open = auto()
+    Mustache = auto()
+    Narrow_Eyes = auto()
+    No_Beard = auto()
+    Oval_Face = auto()
+    Pale_Skin = auto()
+    Pointy_Nose = auto()
+    Receding_Hairline = auto()
+    Rosy_Cheeks = auto()
+    Sideburns = auto()
+    Smiling = auto()
+    Straight_Hair = auto()
+    Wavy_Hair = auto()
+    Wearing_Earrings = auto()
+    Wearing_Hat = auto()
+    Wearing_Lipstick = auto()
+    Wearing_Necklace = auto()
+    Wearing_Necktie = auto()
+    Young = auto()
+
+
+def check_valid_celeba_attr(attr: CelebAttr | str) -> TypeGuard[CelebAttrsEM]:
```

Comment:
This doesn't really make sense, because `CelebAttr` cannot possibly conform to `CelebAttrsEM`.

Reply:
Are you even using this function?

Reply:
No, not anymore. Since I exported CelebA the guard is no longer required - the enum can just be used directly. I've since deleted the function and associated imports.

Reply:
Ah no, I lied. The function was still being used in the 'fair' version of the CelebA DM.